e2e/cypress/integration/menus/status_dropdown_spec.js

```diff
@@ -28,163 +35,106 @@ describe('Status dropdown menu', () => {
         cy.reload();
     });
 
-    it('Displays default menu when status icon is clicked', () => {
-        // # Wait for posts to load
-        cy.get('#postListContent', {timeout: TIMEOUTS.ONE_MIN}).should('be.visible');
-
-        // # Click status menu
-        cy.get('.MenuWrapper .status-wrapper.status-selector button.status').click();
-
-        // # Wait for status menu to transition in
-        cy.get('.MenuWrapper.status-dropdown-menu .Menu__content.dropdown-menu').should('be.visible');
+    after(() => {
+        // # Reset to enable the custom status
+        cy.apiUpdateConfig({TeamSettings: {EnableCustomUserStatuses: true}});
     });
```

Comment:
Instead of resetting in `after` hook, I'd recommend to include it in the predetermined default config ([on-prem](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/support/api/on_prem_default_config.json) and [cloud](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/support/api/cloud_default_config.json)). So that it will be reset in the [global before hook](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress/support/index.js#L165), and we don't need to worry or track similar config change in other (future) spec if needs to reset or not.